code/ship/ship.cpp

```diff
@@ -3665,6 +3674,28 @@ static void parse_ship_values(ship_info* sip, const bool is_template, const bool
 			sip->closeup_zoom = 0.5f;
 		}
 	}
+
+	if(optional_string("$Closeup_pos_targetbox:"))
+	{
+		stuff_vec3d(&sip->closeup_pos_targetbox);
+	} 
+	else 
+	{ 
+		vm_vec_copy_scale(&sip->closeup_pos_targetbox, &sip->closeup_pos, 1.0f);
```

Comment:
Why not copy the vector using assignment, instead of copying and scaling by 1.0?